src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -270,15 +267,15 @@ private ClosePaymentRequestV2Dto.OutcomeEnum authorizationResultToOutcomeV2(
         }
     }
 
-    private Mono<TransactionEvent<TransactionClosureData>> buildClosureEvent(
+    private Mono<TransactionEvent<TransactionClosureData>> processClosureEvent(
```

Comment:
This method does the following things:
 * create a new closure event, either closed or closureFailed
 * decide whether it's enqueueable
 * enqueue it

I would split the method (especially the first two points from the rest), as right now it's not very clear from the outside what it's doing

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/148/commits/e8807b65afebce1c60687eab81d4e543b0efadf7